api/src/domain/updateSidebarCards.test.ts

```diff
@@ -125,6 +125,86 @@ describe("updateRoadmapSidebarCards", () => {
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+    });
+
+    it("doesn't remove the generic welcome card when operating phase is not UP_AND_RUNNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "FORMED_AND_REGISTERED",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome");
+    });
+
+    it("adds the welcome-up-and-running card when operating phase is UP_AND_RUNNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome-up-and-running");
+    });
+
+    it("removes the generic welcome card when operating phase is UP_AND_RUNNING_OWNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+    });
+
+    it("doesn't remove the generic welcome card when operating phase is not UP_AND_RUNNING_OWNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "FORMED_AND_REGISTERED",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome");
+    });
+
+    it("adds the welcome-up-and-running card when operating phase is UP_AND_RUNNING_OWNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome-up-and-running");
+    });
+  });
+
```

Comment:
I think there is a missing test (and behavior) here. What happens when a user "un-graduates" and reverts their phase backwards FROM up-and-running TO formed-and-registered or something else? I assume the welcome card should no longer display the special up-and-running card?